strawberryfields/apps/sample.py

```diff
@@ -86,9 +86,25 @@
 
 The subgraphs sampled from GBS are likely to be dense :cite:`arrazola2018using`, motivating their
 use within heuristics for problems such as maximum clique (see :mod:`~.apps.clique`).
+
+Including node weights
+----------------------
+
+Some graphs are composed of nodes with weights. These node weights can correspond to relevant
+information in an optimization problem and it is desirable to encode them into GBS along with the
+graph's adjacency matrix. One canonical approach to doing this is to use the ``WAW`` encoding
+:cite:`banchi2019molecular`, which rescales the adjacency matrix according to:
+
+.. math::
+
+    A \rightarrow WAW,
+
+with :math:`W` the diagonal matrix formed by the weighted nodes. The rescaled adjacency matrix
```

Comment:
It may be good to write W explicitly. One compact way of doing so is using `W=w_{ij}\delta_{ij}`

Reply:
Nice! Done